### DISCUSSION TITLE
getting Android boot failed, error -22 when flashing sd card via image or manual install

### DISCUSSION THREAD

**Discussion by mariodelgado** [tesla-android/issue-tracker#255] [Q&A]
tried flashing multiple SD cards with latest build, rpi will boot and reboot many times but always end up at the same spot, with a error 

`Android boot failed, error -22`

not sure how to remediate this, seems to happen even with Manual install via fastboot.

**Comment by adamChrystie** [comment]
Try updating pi firmware. [See this link. ](https://www.reddit.com/r/teslaandroid/comments/1354sc3/tesla_android_2023181_is_available/jiq8gua/?utm_source=share&utm_medium=ios_app&utm_name=ioscss&utm_content=1&utm_term=1&context=3)
